arcgis-rest-api/index.d.ts

```diff
@@ -0,0 +1,221 @@
+// Type definitions for arcgis-to-geojson-utils 10.4
```

Comment:
disregard my earlier reply.  

i can see now that @JeffJacobson has contributed both typings for the npm package *and* for generic native types from our REST API.  i agree with the naming of both projects and appreciate both your efforts landing the work here!